digitalpalireader/content/js/send.js

```diff
@@ -304,8 +304,10 @@ function sendPlace(place) {
 	var sidebar = DPRSidebarWindow();
 	if (sidebar) {
 		sidebar.DPRNav.gotoPlace(place);
-	}
-
+  }
```

Comment:
please remove these changes and instead change DPRSidebarWindow so this code works.